app/routes/schemes.js

```diff
@@ -1,7 +1,6 @@
 const { get, post } = require('../api')
 const Joi = require('joi')
 const ViewModel = require('./models/update-scheme')
-const auth = require('../auth')
```

Comment:
Is auth not required for changing payment status?

Reply:
It is, but auth was updated to use route scopes rather than needing a call in the route handler.  This was a remnant that was not removed and caused this route to fail.